api/serializers/day_and_time_slot.py

```diff
@@ -15,3 +15,12 @@ class DayAndTimeSlotSerializer(serializers.ModelSerializer[DayAndTimeSlot]):
     class Meta:
         model = DayAndTimeSlot
         fields = "__all__"
+
+
+class MinifiedDayAndTimeSlotSerializer(serializers.ModelSerializer[DayAndTimeSlot]):
```

Comment:
add a short docstring explaining what kind of minification is done here?

Same for language and level

Reply:
Isn't it clear enough from fields presented inside serializer?

Reply:
Nevertheless, I've added comments

Reply:
Thanks! Well I was more thinking about IDE hints :)